ReactAndroid/src/main/java/com/facebook/react/views/picker/ReactPickerManager.java

```diff
@@ -44,6 +44,11 @@ public void setPrompt(ReactPicker view, @Nullable String prompt) {
     view.setPrompt(prompt);
   }
 
+  @ReactProp(name = "accessibilityLabel")
```

Comment:
One of the internal test failures might be related to this line. Here's what we're seeing internally when I tried to import your changes:

```
stderr: : error: Module com.facebook.react.views.picker.ReactDialogPickerManager has already registered a property named "accessibilityLabel". If you want to override a property, don't add the @ReactProp annotation to the property in the subclass
```

Reply:
I will take a look. I think removing the @ReactProp would fix this according to the log. Thanks for pointing it out!